sphinx/about/whats-new.rst

```diff
@@ -1,6 +1,65 @@
 Version history
 ===============
 
+8.3.0 (2025 August)
+-------------------
+
+New file formats:
+
+* Zeiss XRM:
+   - added support for .txm and .txrm files
+	 
+File format fixes:
+
+* LiFlim
+   - cast integers to double for division accuracy
+* Metamorph
+   - don't reset dimensions if the well count and series count match
+   - fixed the encoding used for reading the .nd file
+* Metamorph TIFF
+   - threw more explicit exceptions if parsing invalid TIFF
+* Nikon ND2
+   - added TextInfoItems to original metadata before parsing
+   - changed the padding for a specific condition to pad to 4 byte boundaries.
+include (thanks to Matthew Smith)
```

Comment:
```suggestion
(thanks to Matthew Smith)
```

Reply:
I think lines 23/24 need to be combined to fix the build.

Reply:
Should be fixed in [85c7ff3](https://github.com/ome/bio-formats-documentation/pull/432/commits/85c7ff31d915075f1a2075d6437952491fa10d6b)